### DISCUSSION TITLE
Is there any way to detect collision of two sprites

### DISCUSSION THREAD

**Discussion by ghost** [pixijs/pixijs#8104] [Q&A]
I'm learning pixi and want to try to make a little game, but that's not the main purpose.

**Comment by domis86** [comment]
Please see previous similar questions:

* https://github.com/pixijs/pixijs/search?q=collision&type=discussions
* https://github.com/pixijs/pixijs/search?q=collision&type=issues

especially:
* https://github.com/pixijs/pixijs/discussions/7605
* https://github.com/pixijs/pixijs/issues/1576
